provider/marathon/builder_test.go

```diff
@@ -33,6 +38,16 @@ func label(key, value string) func(*marathon.Application) {
 	}
 }
 
+func labelWithService(key, value string, serviceName string) func(*marathon.Application) {
+	if len(serviceName) > 0 {
+		property := strings.TrimPrefix(key, types.LabelPrefix)
+		return func(app *marathon.Application) {
+			app.AddLabel(types.LabelPrefix+serviceName+"."+property, value)
+		}
+	}
+	panic("serviceName can not be empty")
```

Comment:
Minor thingie: Please implement this like

```go
if len(serviceName) == 0 {
  panic("serviceName can not be empty")
}
```

One level of indentation less to worry about; and it co-locates the root cause with the error.